test/correctness/CMakeLists.txt

```diff
@@ -169,6 +169,7 @@ tests(GROUPS correctness travis
         inverse.cpp
         isnan.cpp
         issue_3926.cpp
+        issue_4297.cpp
```

Comment:
Nit: can we avoid naming tests this way? It goes against precedent (previous line notwithstanding) and is jarring to read.

Reply:
Yes. Fixed!